### ISSUE TITLE
same code and data occurred reward no change and reward always 0 after ep2

### ISSUE THREAD

**Issue by rcwind** [AI4Finance-Foundation/FinRL#1348]
below is the log:

[@train.py #97] <class 'stable_baselines3.common.vec_env.dummy_vec_env.DummyVecEnv'>
{'buffer_size': 50000, 'learning_rate': 0.0005, 'batch_size': 256, 'action_noise': NormalActionNoise(mu=[0.], sigma=[0.1])}
Using cpu device
Logging to results/ddpg
Episode: 2
[@pyplot.py #497] Loaded backend Agg version v2.2.
day: 241, episode: 2
begin_total_asset: 100000.00
end_total_asset: 100094.35
total_reward: 94.35
total_cost: 2.84
total_trades: 9
Sharpe: 1.463
=================================
Episode: 3
day: 241, episode: 3
begin_total_asset: 100000.00
end_total_asset: 100000.00
total_reward: 0.00
total_cost: 0.00
total_trades: 0
=================================
Episode: 4
day: 241, episode: 4
begin_total_asset: 100000.00
end_total_asset: 100000.00
total_reward: 0.00
total_cost: 0.00
total_trades: 0
=================================
Episode: 5
day: 241, episode: 5
begin_total_asset: 100000.00
end_total_asset: 100000.00
total_reward: 0.00
total_cost: 0.00
total_trades: 0
=================================
 how this would happen when code and data is the same?
 some times is ok when train.

**Comment by zhumingpassional** [comment]
psl check your datasets, params
